joplin.edn

```diff
@@ -0,0 +1,7 @@
+{:migrators    {:sql-mig   "resources/migrations/"}
+
+ :seeds        {:sql-seed  "seeds.sql/run"}
+ 
+ :databases    {:psql-dev  {:type :sql, :url  ["postgresql:postgres@localhost:5432/postgres"]}}
```

Comment:
* move the file to `resources`
* change `["postgresql:postgres@localhost:5432/postgres"]` to `"jdbc:postgresql:postgres@localhost:5432/postgres"` (unwrap vector, add jdbc)

Reply:
Uhm, that string didn't work for me.

`"jdbc:postgresql:postgres"` works well.